cloud/deployment/deployment.go

```diff
@@ -328,6 +341,40 @@ func selectCluster(clusterID, organizationID string, client astro.Client) (newCl
 	return clusterID, nil
 }
 
+func healthPoll(deploymentID, ws string, client astro.Client) error {
+	fmt.Printf("Waiting for the deployment to become healthy…\n\nThis may take a few minutes\n")
+	time.Sleep(time.Duration(sleepTime) * time.Second)
+	buf := new(bytes.Buffer)
+	timeout := time.After(time.Duration(timeoutNum) * time.Second)
+	ticker := time.NewTicker(time.Duration(tickNum) * time.Second)
+	for {
+		select {
+		// Got a timeout! fail with a timeout error
+		case <-timeout:
+			return errTimedOut
+		// Got a tick, we should check if docker is up & running
+		case <-ticker.C:
+			buf.Reset()
+			deployments, err := getDeployments(ws, client)
```

Comment:
nothing related to this PR, but long term having a `getDeployment` API, which can give us a deployment info based on ID passed to the API would be really useful.

Reply:
Agreed. I noticed this in few other places too. Can we file a ticket for this so we dont forget. The endpoint does exists today, we just need to add the same in the cli